src/utils/theme/index.js

```diff
@@ -69,10 +80,9 @@ export const COLORS = {
   black75: 'rgba(0, 0, 0, 0.75)',
   transparent: 'transparent',
   anonSecondary20: 'rgba(0, 173, 181, 0.2)',
-  gray400: '#9B9FA9',
   lightgrey60: 'rgba(245, 246, 247, 0.6)',
-  gray300: '#D2D4DB',
-  gray200: '#E8EBED'
+  downvote: '#E2887E',
+  upvote: '#69DD10'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color `downvote` is a shade of red and `upvote` is green. These colors are intuitive as they follow a common pattern (red for negative, green for positive). However, ensure that these colors are accessible and can be easily distinguished by color-blind users.


<!-- This is an auto-generated comment by OSS CodeRabbit -->